src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
```

Comment:
Sure mate...not everything is perfect :) ...but we should all discuss how many and what changes to pullmin for a given ticket to make it clear and consistent :). I am not saying no to the refactorings but please note that if you change everything you touch (as someone suggested us to do so) then it is very hard to read changes